RuntimeCrashTests/run-runtime-crash-tests.sh

```diff
@@ -0,0 +1,46 @@
+echo "Running Runtime Crash tests\n"
```

Comment:
Add a shebang & pipefail
```suggestion
#!/usr/bin/env bash
set -euo pipefail

echo "Running Runtime Crash tests\n"
```

Reply:
pipefail will make the script fail. 
The way it currently is set up it reads how it failed and then responds. Did it this way since we expect certain things to fail.

Reply:
You're right, missed the exit code mangling